reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -63,6 +63,10 @@
  * Factories prefixed with {@code new} (eg. {@link #newBoundedElastic(int, int, String)} return a new instance of their flavor of {@link Scheduler},
  * while other factories like {@link #boundedElastic()} return a shared instance - which is the one used by operators requiring that flavor as their default Scheduler.
  * All instances are returned in a {@link Scheduler#init() initialized} state.
+ * <p>
+ * Since 3.6.0 {@link #boundedElastic()} can run tasks on {@link VirtualThread}s if the application
+ * runs on a Java 21 runtime and the {@link #DEFAULT_BOUNDED_ELASTIC_ON_VIRTUAL_THREADS}
```

Comment:
Please revisit all the places where we mention JDK 21 and consider if 21+ is more appropriate